### ISSUE TITLE
accessibility view does not work on linux 

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/vscode#186437]
Testing #186214

![image](https://github.com/microsoft/vscode/assets/3840081/91e4ea57-c30f-4362-8ba5-1e9daa08fe2f)

In the above image, it only reads: `(method) def has_id(` rest of the content is skipped.

**Comment by meganrogge** [comment]
That is as designed - it will read only the line the cursor is on. Screen reader users will know how to navigate down.

**Comment by karthiknadig** [comment]
@meganrogge this is in the hover, I did not see a way to move to the next line in the hover view.

**Comment by meganrogge** [comment]
In your screenshot,the accessible view isn't open - maybe due to the key binding issue.

**Comment by karthiknadig** [comment]
I could not trigger it at all even with clicking on the command from the palette

**Comment by meganrogge** [comment]
Really strange as all of this works on macOS and probably windows

**Comment by meganrogge** [comment]
@karthiknadig if you assign the command some keybinding which is not taken already, does it work? any errors in dev tools console?

**Comment by karthiknadig** [comment]
It works!! i assigned `ctrl+shift+4` for this and it works as expected.

**Comment by meganrogge** [comment]
awesome, thanks @karthiknadig so I'll close this and work on the keybinding issues you've created

**Comment by rzhao271** [comment]
Verified that it works with Ctrl+Shift+4 on Ubuntu. The OS is probably taking Alt+F2.
